src/Makefile

```diff
@@ -165,10 +165,12 @@ docker-publish: docker
 
 clean:
 	$(RM) metadatad $(OBJS)
+	(cd ../test && $(MAKE) clean)
```

Comment:
Should 'make clean' in the test/ subdir also clean up the src/ dir?

Reply:
I'd like to promote using `make test` from the src directory, so leaving as-is for now.